Plugins/SafeDIGenerator/SafeDIGenerateDependencyTree.swift

```diff
@@ -11,45 +11,85 @@ struct SafeDIGenerateDependencyTree: BuildToolPlugin {
             return []
         }
 
-        let outputSwiftFile = context.pluginWorkDirectory.appending(subpath: "SafeDI.swift")
```

Comment:
changes to this file are best viewed with whitespace changes turned off. Apple deprecated a _ton_ of the path-based APIs in Swift 6 / Xcode 16.